docs/evaluation/tutorials/backtesting.mdx

```diff
@@ -4,118 +4,169 @@ sidebar_position: 5
 
 # Backtesting
 
-Deploying your app into production is just one step in a longer journey continuous improvement. You'll likely want to develop other candidate systems that improve on your production model using improved prompts, llms, indexing strategies, and other techniques. While you may have a set of offline datasets already created by this point, it's often useful to compare system performance on more recent production data.
+Deploying your application is just the beginning of a continuous improvement process.
+Typically, you'll want to refine your system by enhancing prompts, language models, tools, and architectures.
+While the initial datasets you created before deployment are useful for testing new versions, you now have access to real historical data. This data provides a more accurate representation of the inputs your application will encounter in the future.
+Backtesting involves assessing new versions of your application using this historical data and comparing the new outputs to the original ones.
+This approach can offer a clearer indication of whether the new version is an improvement over the current deployment compared to evaluations using pre-production datasets.
 
-This notebook shows how to do this in LangSmith.
+Here are the basic steps for backtesting:
 
-The basic steps are:
+1. Select sample runs from your production tracing project to test against.
+2. Transform the run inputs into a dataset and record the run outputs as an initial experiment against that dataset.
+3. Execute your new system on the new dataset and compare the results of the experiments.
 
-1. Sample runs to test against from your production tracing project.
-2. Convert runs to dataset + initial experiment.
-3. Run new system against the dataset to compare.
+This process will provide you with a new dataset of representative inputs, which you can version and use for backtesting your models.
 
-You will then have a new dataset of representative inputs you can you can version and backtest your models against.
+:::info Ground truth data
+Often, you won't have definitive "ground truth" answers available.
+In such cases, you can manually label the outputs or use evaluators that don't rely on reference data.
+If your application allows for capturing ground-truth labels, for example by allowing users to leave feedback, we strongly recommend doing so.
+:::
 
```

Comment:
Add a sentence here about what the tutorial covers. eg In this tutorial, we will go over XYZ..